prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_AOE_1_20230220-0001.hl7

```diff
@@ -70,7 +70,7 @@ OBX|6|ST|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMME
 OBX|7|ST|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMMENT^L|1.1|Staphylococcus epidermidis   Streptococcus anginosus group||||||F||||RML^TMCA, SOUTH CAMPUS^LB||||20210805003800-0700||||^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^06D0055551|1501 S. POTOMAC^^AURORA^IG^80012^USA^B^^MD|1740336429^KOTNIS^GREGORY^R^MD^DR.^^^RML^L^^^FI^TMCA, SOUTH CAMPUS
 OBX|8|ST|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMMENT^L|1.1|Staphylococcus lugdunensis   Streptococcus faecalis||||||F||||RML^TMCA, SOUTH CAMPUS^LB||||20210805003800-0700||||^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^06D0055551|1501 S. POTOMAC^^AURORA^IG^80012^USA^B^^MD|1740336429^KOTNIS^GREGORY^R^MD^DR.^^^RML^L^^^FI^TMCA, SOUTH CAMPUS
 OBX|9|ST|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMMENT^L|1.1|Streptococcus spp.           Enterococcus faecium||||||F||||RML^TMCA, SOUTH CAMPUS^LB||||20210805003800-0700||||^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^06D0055551|1501 S. POTOMAC^^AURORA^IG^80012^USA^B^^MD|1740336429^KOTNIS^GREGORY^R^MD^DR.^^^RML^L^^^FI^TMCA, SOUTH CAMPUS
-OBX|10|ST|77202-0^Laboratory comment [Text] in Report Narrative^LN^VERGPCOM^COMMENT^L|1.1|Streptococcus pneumoniae     ||||||F||||RML^TMCA, SOUTH CAMPUS^LB||||20210805003800-0700||||^L^^^^CLIA&2.16.840.1.113883.4.7&ISO^XX^^^06D0055551|1501 S. POTOMAC^^AURORA^IG^80012^USA^B^^MD|1740336429^KOTNIS^GREGORY^R^MD^DR.^^^RML^L^^^FI^TMCA, SOUTH CAMPUS
```

Comment:
Do we not want the spare whitespace to make sure that it's trimming the way we expect?

Reply:
Looks like it comes from the `String.trimAndTruncate` function that trims the string regardless if it got truncated or not. If thats not what we want we'll need to devise a way to skip over that logic for fields that we want as is.

Reply:
note to me: make no changes to any int test files